packages/cozy-stack-client/src/CozyStackClient.js

```diff
@@ -133,6 +133,12 @@ class CozyStackClient {
     }
   }
 
+  onTokenRefresh(token) {
```

Comment:
Should we add the docstring?

Reply:
It's supposed to be private so I am OK not adding a docstring here.